nuxeo-features/nuxeo-platform-rendition/nuxeo-platform-rendition-core/src/main/resources/OSGI-INF/rendition-operations-contrib.xml

```diff
@@ -6,6 +6,8 @@
 
     <operation class="org.nuxeo.ecm.platform.rendition.operation.GetRendition" />
     <operation class="org.nuxeo.ecm.platform.rendition.operation.GetContainerRendition" />
+    <operation class="org.nuxeo.ecm.platform.rendition.operation.PublishRendition"  replace="true"/>
```

Comment:
I'm not sure it's wise to have an operation that replaces another. Why is that needed?